application/app/src/main/resources/application.yaml

```diff
@@ -1,13 +1,25 @@
+# This file is required for running the API in VSC
+spring:
+  elasticsearch:
+    rest:
+      uris: "localhost:9200"
+
 server:
-  port: ${SERVER_PORT}
+  port: 9095
```

Comment:
Are we reverting back to the hardcoded variables?

Reply:
No, its just I couldn't get it working running through VSC with placeholders.  I'll get back around to looking at this, it's on my list...